src/wh_server_she.c

```diff
@@ -311,7 +353,10 @@ static int hsmSheLoadKey(whServerContext* server, whPacket* packet,
     uint8_t kdfInput[WOLFHSM_SHE_KEY_SZ * 2];
     uint8_t cmacOutput[AES_BLOCK_SIZE];
     uint8_t tmpKey[WOLFHSM_SHE_KEY_SZ];
-    whNvmMetadata meta[1];
+    whNvmMetadata meta[1] = {{0}};
```

Comment:
Doesn't this just waste cycles? We don't read the metadata before writing to it.